test/cli/job/run-fail/expected-stdout

```diff
@@ -0,0 +1,2 @@
+Starting 1 job
+Waiting for %%TEST_BRANCH_MAIN_ID%%/keboola.runner-config-test/%%TEST_BRANCH_MAIN_CONFIG_RUNNER_FAIL_ID%% (0)%A
```

Comment:
Ten output není deterministický (těch `Waiting for ...` zpráv tam bude víc pokud job trvá víc jak 5 sekund), snad tady bude stačit `%A` a nebude z toho další flaky test.

Reply:
Tak tam daj namiesto celeho `Waiting ...` `%A`.
Skor mi tu chyba nejaka sprava na konci, ze job skoncil uspesne.

Reply:
Resp. uspesne to mas dalej, ale neuspesne + chybova hlaska.

Reply:
Chybova hláška je v `expected-stderr`. Chtěl jsem tam dát aspoň jeden ten `Waiting ...`, aby se otestoval celý ten output.